ingenialink/ethercat/servo.py

```diff
@@ -71,17 +71,15 @@ class EthercatServo(PDOServo):
         dtype=REG_DTYPE.S32,
         access=REG_ACCESS.RW,
     )
-    RPDO_ASSIGN_REGISTER_SUB_IDX_1 = [
-        EthercatRegister(
-            identifier="RPDO_ASSIGN_REGISTER",
-            units="",
-            subnode=0,
-            idx=0x1C12,
-            subidx=0x01,
-            dtype=REG_DTYPE.S32,
-            access=REG_ACCESS.RW,
-        )
-    ]
+    RPDO_ASSIGN_REGISTER_SUB_IDX_1 = EthercatRegister(
```

Comment:
Why is this not a list anymore?
If by using complete access we are using more than the declared subidx here this is really confusing.
Maybe it's time we create a "SubindexableEthercatRegister" class that represents registers that have multiple subindexes

Reply:
I think it makes no sense that it was a list, we only have one RPDO_ASSIGN_REGISTER_SUB_IDX_1 and it can't be more. If you want a list element for each subindex then the name should be RPDO_ASSIGN_REGISTER.
About "SubindexableEthercatRegister", yes, probably we will need something like that we can use the new dictionary definition to think about it.

Reply:
Well it did make sense if each subidx had a definition. 
Since we only contemplated one pdo map assignment a list with one item was the correct choice.
Now this change is confusing, I would add a comment explaining that this register is representing all the subindexes by using complete access later, and that will be replaced with the "SubindexableEthercatRegister" when it's implemented.
And probably remove SUB_IDX_1 from the name

Reply:
Technically it is just the sub-index 1, it is indeed used for complete access, but it does not affect this variable.